app/foodrle/game.py

```diff
@@ -224,14 +224,14 @@ def calorie_hint(answer: Dishes, guess_dish: Dishes):
     return res
 
 
-def taste_hint (answer: Taste, guess_dish: Taste):
+def taste_hint(answer: Taste, guess_taste: Taste):
     res = {}
     ans_dict = answer.__dict__
-    guess_dict = guess_dish.__dict__
-    
+    guess_dict = guess_taste.__dict__
+
     for k, v in guess_dict.items():
-        if v == True: 
-            if ans_dict.get(k) == True:
+        if v:
+            if ans_dict.get(k):
```

Comment:
should be == True (removing introduces a bug)
https://user-images.githubusercontent.com/96888069/202963895-13f1fd0f-e0b2-43c4-ae81-5665fa3fbade.png